cmd/builder/internal/builder/main.go

```diff
@@ -144,7 +155,7 @@ func GetModules(cfg Config) error {
 	}
```

Comment:
`go mod tidy` is run a few lines above and downloads the needed dependencies, so `go mod download` isn't needed. No dependencies are updated after the tidy call.

Reply:
Thanks, this comment was useful!